### ISSUE TITLE
Create releases based on the latest tags

### ISSUE THREAD

**Issue by tobon4** [google-github-actions/release-please-action#960]
### TL;DR

We have our own tagging scheme, and would like release-please to create releases and PRs based on our own tags. 

### Detailed design

Today, we have a scheme (a bit similar to conventional commits), but we check for specific words in the PR title, such as "major", "minor", and "patch". 

Once the pull request is merged, we create the tag. 

It seems release-please only keeps track of what releases were created. So, for example, if we have release 1.2.3, but we have now merged 10 pull requests, so we are on tag 1.3.9 - for example - we'd like release-please to create a release based on that tag version, and not 1.3.0. 

Something like: 
`create-release-based-on-latest-tag: true`

Or if there's some other way to tell release-please to adhere to this pattern. 

I've only found `bump-minor-pre-major`, `ump-patch-for-minor-pre-major`, `include-v-in-tag`, and `include-component-in-tag` but don't think this solves our use case.
